lib/DI/ContainerConfigurator.php

```diff
@@ -88,6 +89,8 @@ function configure(ContainerBuilder $container) {
     $container->autowire(\MailPoet\Segments\WooCommerce::class)->setPublic(true);
     // Settings
     $container->autowire(\MailPoet\Settings\SettingsController::class)->setPublic(true);
+    // User Flags
+    $container->autowire(\MailPoet\Settings\UserFlags::class)->setPublic(true);
```

Comment:
I think this doesn't have to be set to public.

Reply:
Can you tell me why? I see that all others are set to public. I am still new to DI so I don't really know what is the difference.

Reply:
It is explained here: https://symfony.com/doc/2.4/components/dependency_injection/advanced.html#marking-services-as-public-private

And they even said it is a bad practice to have services public: https://symfony.com/blog/new-in-symfony-3-4-services-are-private-by-default